aktualizr-dev.rb

```diff
@@ -0,0 +1,37 @@
+class AktualizrDev < Formula
+  desc "C++ Client for HERE OTA Connect"
+  homepage ""
+  version = "2020.2"
+  url "https://github.com/advancedtelematic/aktualizr.git", :using => :git, :tag => "#{version}"
+  sha256 "782fa343c85be455d6e51bd774f3244e0dad093989ac9bb1d96215785f7e7314"
```

Comment:
For this one, wouldn't it make more sense to build master?

Reply:
Actually, come to think of it, homebrew has a `--HEAD` option. We could just add `head "https://github.com/advancedtelematic/aktualizr.git"` to the main formula, and then do `brew install --fetch-HEAD --HEAD aktualizr` if/when we want to test the master build.

Reply:
My initial intention was to build a head of branch and this `head` feature didn't work for branch for me.

Reply:
Okay. I still don't understand what this formula is for; after all, it's building from a tagged release just like the main aktualizr recipe. But if you find it useful, I don't see any reason not to have it here.